src/api/models/protocols/benthic.py

```
@@ -0,0 +1,374 @@
+    )
+
+    class Meta:
+        db_table = "obs_benthic_photo_quadrat"
+        verbose_name = _("benthic photo quadrat transect observation")
+        unique_together = (
+            "benthic_photo_quadrat_transect",
+            "quadrat_number",
+            "attribute",
+            "growth_form",
+        )
+        ordering = ["created_on"]
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)